packages/frontend/package.json

```diff
@@ -12,10 +12,13 @@
   "devDependencies": {
     "@nuxt/devtools": "latest",
     "nuxt": "^3.8.1",
+    "sass": "^1.72.0",
     "vue": "^3.3.8",
     "vue-router": "^4.2.5"
   },
   "dependencies": {
+    "bootstrap": "^5.3.3",
+    "bootstrap-vue": "^2.23.1",
```

Comment:
Bootstrap vue should not be in the package file. You can remove this by uninstalling it via the terminal

Reply:
Good catch. I have fixed this.